.gitlab/ci/bucket-upload.yml

```diff
@@ -0,0 +1,165 @@
+    - section_start "Install Packs"
+    - ./Tests/Marketplace/install_packs.sh "$INSTANCE_ROLE"
+    - section_end "Install Packs"
+    - section_start "Destroy Instances"
+    - |
+      if [ -n "${CONTRIB_BRANCH}" ]; then
+        echo "Skipping - not running in contribution instance"
+      else
+        python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ENV_RESULTS_PATH "$INSTANCE_ROLE" "$TIME_TO_LIVE"
+        export PSWD=$(jq .serverLogsZipPassword < $(cat secret_conf_path) | cut -d \" -f 2)
+        zip -P $PSWD -j $ARTIFACTS_FOLDER/Logs.zip $ARTIFACTS_FOLDER/server*.log || ((($? > 0)) && echo "Didn’t find any server logs, skipping this stage" && exit 0)
+        rm -f $ARTIFACTS_FOLDER/server*.log
+      fi
+    - section_end "Destroy Instances"
+
+install-packs-in-server6_0:
+  extends: .install_packs_in_server
+  variables:
+    INSTANCE_ROLE: "Server 6.0"
+
+install-packs-in-server-master:
+  extends: .install_packs_in_server
+  variables:
+    INSTANCE_ROLE: "Server Master"
+
+upload-packs-to-marketplace:
+#  needs: ["run-validations-upload-flow", "run-unittests-and-lint-upload-flow", "install-packs-in-server6_0", "install-packs-in-server-master"]
+  needs: ["run-validations-upload-flow", "install-packs-in-server6_0", "install-packs-in-server-master"]
+  stage: upload-to-marketplace
+  artifacts:
+    expire_in: 48 hrs
+    paths:
+      - /builds/xsoar/content/artifacts/*
+  variables:
+    INSTANCE_ROLE: "Server Master"
+    INSTANCE_CREATED: "true"
+    SSH_TUNNEL_TIMEOUT: 10
+    TIME_TO_LIVE: ""
+  extends:
+    - .bucket-upload-rule
+    - .default-job-settings
+  script:
+    - *start_tunnel
+    - *check_user_permissions_to_upload_packs
+    - section_start "Upload Packs To Marketplace Storage"
+    - |
+      if [[ "$CI_COMMIT_BRANCH" == "master" ]] || [[ "$GCS_MARKET_BUCKET" != "$GCS_PRODUCTION_BUCKET" ]]; then
+        EXTRACT_FOLDER=$(mktemp -d)
+        PACK_ARTIFACTS=$ARTIFACTS_FOLDER/content_packs.zip
+        PACKS_DEPENDENCIES=$ARTIFACTS_FOLDER/packs_dependencies.json
+        CI_COMMIT_BRANCH=${CI_COMMIT_BRANCH:-unknown}
+        GCS_BUILD_BUCKET="marketplace-ci-build"
+        if [[ $GCS_MARKET_BUCKET != $GCS_PRODUCTION_BUCKET ]]; then
+          STORAGE_BASE_PATH="upload-flow/builds/$CI_COMMIT_BRANCH/$CI_PIPELINE_ID/content/packs"
+        fi
+        python3 ./Tests/Marketplace/copy_and_upload_packs.py -a $PACK_ARTIFACTS -e $EXTRACT_FOLDER -pb "$GCS_MARKET_BUCKET" -bb "$GCS_BUILD_BUCKET" -s $GCS_MARKET_KEY -n $CI_PIPELINE_ID -c $CI_COMMIT_BRANCH -pbp "$STORAGE_BASE_PATH"
+      fi
+    - section_end "Upload Packs To Marketplace Storage"
+    - section_start "Validate Premium Packs"
+    - |
+      if [[ "$CI_COMMIT_BRANCH" == "master" ]] || [[ $GCS_MARKET_BUCKET != $GCS_PRODUCTION_BUCKET ]]; then
+        ./Tests/scripts/validate_premium_packs.sh "$INSTANCE_ROLE"
+      else
+        echo "Skipping Premium Packs Validation"
+      fi
+    - section_end "Validate Premium Packs"
```

Comment:
I thought we wanted to upload to a different bucket when it was `marketplace-dist-dev` but here it seems we are just skipping it?

Reply:
I replaced marketplace-dist with $GCS_PRODUCTION_BUCKET setting it to marketplace-dist in circle and to marketplace-dist-dev in gitlab